modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -393,7 +393,11 @@ def buildProject(sassLint = true) {
         [$class: 'StringParameterDefinition',
           name: 'SCHEMA_BRANCH',
```

Comment:
Potentially yes, but it'll be messy if we try to do that before all the dependent apps are moved over to use buildProject so probably continue with both params until then.